teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<p>Maybe instead of</p><blockquote>Show the compounds with images.</blockquote><p>Something more descriptive like</p><p>"Show query and similar compounds in 2D structure representation"</p><p>?</p><p> </p><p>Maybe expose <code>"Gefitinib"</code> also to the function signature? (In that case, if we changed the input query and similarity results, the function would still work).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>done</p>